R/hms_class.R

```diff
@@ -270,4 +270,104 @@ setMethod("printBlockedSprouts", "hms", function(object) {
   }
 })
 
+#' Title
+#'
+#' @param object - hms s4 object
+#' @param dimensions - two selected dimensions
+#'
+#' @export
+setGeneric("plotPopulation", function(object, dimensions) standardGeneric("plotPopulation"))
+
+#' Title
+#'
+#' @param object - hms s4 object
+#' @param dimensions - two selected dimensions
+#'
+#' @export
+setMethod("plotPopulation", "hms", function(object, dimensions) {
+  dimensions_names <- mapply(function(n) {
+    paste("x", toString(n), sep = "")
+  }, dimensions)
```

Comment:
To jest po prostu `c("x1", "x2")`, tak? Czytelniej może by to było wprost napisać zamiast tego mapply

Reply:
no zależnie od tego jakie wymiary poda użytkownik xd jak poda c(3,5) to będzie "x3" i "x5" 
chodzi Ci o to żeby pokazywać explicite że to są 2 zmienne?
```
first_dimension_name <- paste("x", toString(dimensions[[1]]), sep = "")
second_dimension_name <- paste("x", toString(dimensions[[2]]), sep = "")
```

Reply:
aa, oki, nie zrozumiałem tego `dimensions`